models/sales/sales.yml

```diff
@@ -59,7 +59,60 @@ models:
       - name: owner_full_name
         description: The full name of the deal's owner.
 
-        
+  - name: hubspot__deal_stages
+    description: Each record represents a stage within a deal in Hubspot
+    columns:
+      - name: deal_stage_id
+        description: The unique deal stage identifier.
+        tests:
+          - not_null
+          - unique
```

Comment:
i think this might not always pass since we're pulling in `deal_company`, which has a 1:n relationship

Reply:
maybe we shouldn't bring the company into the `deal_stages` model, which is what we decided with `hubspot__deals` (and users can just join with `deal_company` and `company` using the staging models)

Reply:
Ahh I forgot about that issue. That is a great point. I will remove the deal_company and refer the customer that they can simply join company in if they would like. Thanks!